config/routes.rb

```diff
@@ -16,6 +16,7 @@
   namespace :api do
     namespace :v1 do
       resources :recipe
+      resources :user
      
```

Comment:
NitPick: I think this newline is unnecessary.

Reply:
deleted